.github/workflows/ci.yml

```diff
@@ -966,14 +968,38 @@ jobs:
             --env "BUILD_PRESET=benchmark-with-tracing" \
             --env "IREE_HOST_BIN_DIR=${BUILD_DIR}/install/bin" \
             "${DOCKER_IMAGE}" "${BUILD_SCRIPT}" "${BUILD_TOOLS_DIR}/build-traced"
+      - name: "Downloading pre-built tracy capture tool"
+        id: download-tracy-capture
+        env:
+          TRACY_CAPTURE_GCS_ARTIFACT: ${{ matrix.target.tracy_capture }}
+          TRACY_CAPTURE: ${{ env.BUILD_TOOLS_DIR }}/build-traced/tracy-capture
+        run: |
+          gcloud storage cp "${TRACY_CAPTURE_GCS_ARTIFACT}" "${TRACY_CAPTURE}"
+          chmod +x "${TRACY_CAPTURE}"
+          echo "tracy-capture=${TRACY_CAPTURE}" >> "${GITHUB_OUTPUT}"
       - name: "Creating the benchmark tools archive"
+        # Here we pack a tracy-capture binary into each benchmark tools archive.
+        # This could be wasteful because multiple benchmark tools archives might
```

Comment:
I'm still not really thrilled with this approach. I think it would be useful to reference how big these binaries are and compare that to the other stuff we're passing around. I guess this is fine for now if it's small. It seems like the host itself could etermine what capture tool it needs without that being part of the benchmark configuration.

Reply:
Docs the binary size (~7MB)

Reply:
It's around 30% of the total benchmark tools archive
```sh
tar -tvf linux-x86_64-benchmark-tools.tar 
-rwxr-xr-x runner/runner 10405288 2023-03-28 17:12 linux-x86_64-benchmark-tools-dir/build-traced/tools/iree-benchmark-module
-rwxr-xr-x runner/runner  8380576 2023-03-28 17:12 linux-x86_64-benchmark-tools-dir/build/tools/iree-benchmark-module
-rw-r--r-- runner/runner      465 2023-03-28 17:12 linux-x86_64-benchmark-tools-dir/build-traced/tools/build_config.txt
-rw-r--r-- runner/runner      465 2023-03-28 17:12 linux-x86_64-benchmark-tools-dir/build/tools/build_config.txt
-rwxr-xr-x runner/runner  7059904 2023-03-28 17:12 linux-x86_64-benchmark-tools-dir/build-traced/tracy-capture
```